File: UTESA/DCE 5 Estrellas/dce2005/Estrella1/DCE1_Ejemplos/src/VB/Cliente/OpcionInvalidaException.vb
''' <summary>
''' Representa un error que ocurre cuando se selecciona una opción inválida.
''' </summary>
Public Class OpcionInvalidaException
    Inherits ApplicationException

    ''' <summary>
    ''' Construye una instancia en base a un mensaje de error.
    ''' </summary>
    ''' <param name="mensaje">El mensaje de error.</param>
    Public Sub New(ByVal Mensaje As String)
        MyBase.New(Mensaje)
    End Sub

End Class

File: UTESA/DCE 5 Estrellas/dce2005/Estrella1/DCE1_Ejemplos/src/CS/Cliente/OpcionInvalidaException.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DCE05.Ejemplos.EstrellaUno.Cliente {

    /// <summary>
    /// Representa un error que ocurre cuando 
    /// se selecciona una opción inválida.
    /// </summary>
    class OpcionInvalidaException : ApplicationException {

        /// <summary>
        /// Construye una instancia en base a un mensaje de error.
        /// </summary>
        /// <param name="mensaje">El mensaje de error.</param>
        public OpcionInvalidaException(string mensaje) : base(mensaje) { }
    }
}
